src/MapViewController.swift

```diff
@@ -504,17 +507,17 @@ extension MapViewController : TGMapViewDelegate, TGRecognizerDelegate {
   
   open func mapView(_ mapView: TGMapViewController, didSelectFeature feature: [AnyHashable : Any]?, atScreenPosition position: CGPoint) {
     guard (feature != nil) else { return }
-    featureSelectDelegate?.mapController(self, didSelectFeature: feature, atScreenPosition: position)
+    featureSelectDelegate?.mapController(self, didSelectFeature: feature!, atScreenPosition: position)
```

Comment:
Nitpick: this will work but we can do better! 

Change the above guard statement to be:
`guard let feature = feature else { return }`

And then you can use the shadowed unwrapped optional in the call to the delegate, and thus avoid force unwrapping :)